packages/medusa/src/api/routes/admin/currency/index.ts

```diff
@@ -0,0 +1,37 @@
+import { Router } from "express"
```

Comment:
nit: would the parent folder of this file to be renamed `currency -> currencies` to follow the convention